client/assets/js/draw.class.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import Snake from './snake.class';
```

Comment:
You don't need to call it `.class`. `snake` and the uppercase `Snake` import are enough indicators. Please fix this before I merge